json-specs/input-spec.json

```diff
@@ -1233,6 +1242,24 @@
         "type": "bool",
         "doc": "Whether to use the convergent (area weighted) formulation of IPC."
     },
+    {
+        "pointer": "/contact/use_area_weighting",
+        "default": true,
```

Comment:
Yes. I haven't actually made it possible for the user to turn on only parts of the convergent formulation. I was hesitating because I wasn't sure what the use case would be and didn't want to add options that won't be used to the spec. Do you have any thoughts about this? I can go ahead and add those options (while preserving default behavior) or remove them from the spec.